pygamelib/pygamelib.js

```diff
@@ -147,6 +158,7 @@ var PygameLib = {};
         mod.get_ticks = new Sk.builtin.func(function() {
             return Sk.ffi.remapToPy(new Date() - PygameLib.initial_time);
         });
+        // TODO: mod.delay
```

Comment:
I guess the same implementation as time.wait is okay, because in both cases we have to pause Skulpt from continuing interpreting the code until delay is over.